### DISCUSSION TITLE
How do you mix the two transforms from different package

### DISCUSSION THREAD

**Discussion by one-matrix** [Project-MONAI/MONAI#6125] [Q&A] [answered]
for example：
```
preprocess=Compose([
    monai.transforms.EnsureChannelFirst(),
    torchvision.transforms.ToTensor(),
])
```

**Comment by KumoLiu** [comment, marked as answer]
Hi @one-matrix, thank you for your interest here.

- You could use `TorchVision`, a wrapper transform for PyTorch TorchVision transform based on the specified transform name and args.
https://github.com/Project-MONAI/MONAI/blob/354c9c2be3139b1654811784340d094f78a5c7d6/monai/transforms/utility/array.py#L1153-L1159

- You could also write a custom transform to do such things, for more details refer to this [discussion](https://github.com/Project-MONAI/MONAI/discussions/4809).
Hope it can help you, thanks!

**Reply by one-matrix** [reply]
thank @KumoLiu  .This is very useful. Thank you for your answer.
